src/DetailsView/components/issues-table.tsx

```diff
@@ -27,6 +27,7 @@ export type IssuesTableDeps = IssuesDetailsPaneDeps & ExportDialogDeps;
 export interface IssuesTableProps {
     deps: IssuesTableDeps;
     title: string;
+    subtitle?: JSX.Element;
```

Comment:
Does `subtitle` need to be optional? The code builds successfully without the `?`.

Reply:
That's because our codebase doesn't enforce strict null checks. I think it's more clear for it to be explicit about being optional, though (the visualizers for the non-issues cases intentionally don't set it)